packages/childchain/README.md

```diff
@@ -1,14 +1,39 @@
 # Running the node
 
-To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed: 
+To start the node make sure to have ethermint (v0.1.0 or higher) installed and have your GO Path exposed:
 
 ```
 git clone https://github.com/ChainSafe/ethermint
 cd ethermint
 make install
 ```
-To run the chain run `./childchain.sh` and in a second terminal/tab `ethermintcli rest-server --laddr "tcp://localhost:8545" --unlock-key mykey --chain-id 8 --trace`
 
 # Run the tests
 
 To compile the contracts run `truffle build`, to run the tests run `truffle test`.
+
+# Start-up script
+
+The underlying bash-Script `childchain.sh` starts the Ethermint chain, initializes a genesis account as well as starting a RPC server. To stop the chain the command `pkill ethermint` should be used or analogous command.
+
+Requirements to run this script are the installation of Ethermint, truffle, node.js, perl and sed.
+
+After the Ethermint chain is started, a genesis deployment account is used to deploy the Bridge contract using `truffle migrate`.
```

Comment:
Would be nice to have a more clear description of the deployment sequence, at least on the high level. Also a short description of how it will be connected to the main chain (even if it's not a part of the script)

Reply:
further information on the childchain setup can be found in the appropriate docs folder, the deployment for the production version is described below